scripts/redact_cli_py/instruction.md

```diff
@@ -0,0 +1,102 @@
+# How to use the Redaction Tool?
+
+## Overview
+The Redaction Tool is provided by Azure Form Recognizer team to help our customers who are willing to share/donate data with Microsoft that can share/donate data efficiently without privacy concern. It is focused on Personally Identifiable Information (PII) or sensitive data labeling and redaction while keeping the semantics of these fields (e.g. length of the values, character/digit patterns, upper/lower case etc.) for prebuilt model training. This is a step-by-step instruction to guide our you how to use the tool.
+
+![process-overview](./images/redaction-tool.png)
+
+## Language support
+This tool supports Latin characters redaction only. For any non-Latin document support, please [contact us](mailto:formrecog_contact@microsoft.com?subject=Redaction%20tool%20language%20support).
+
+## Prerequisites
+- Azure subscription - [Create one for free](https://azure.microsoft.com/free/cognitive-services)
+- Once you have your Azure subscription, create a [Form Recognizer resource](https://ms.portal.azure.com/#create/Microsoft.CognitiveServicesFormRecognizer) in the Azure portal to get your key and endpoint.
+- Create an Azure storage account with a container - following the [Azure Storage quickstart for Azure portal](https://docs.microsoft.com/en-us/azure/storage/blobs/storage-quickstart-blobs-portal). Use the *standard* performance tier.
+- [Optional] If the data cannot leave your environment, you need to follow [this document](https://docs.microsoft.com/en-us/azure/applied-ai-services/form-recognizer/label-tool#set-up-the-sample-labeling-tool) to set up the labeling tool container.
+
+## Step 1 - label the PII/sensitive fields with Sample Labeling Tool (FOTT)
+* Assemble your raw images based on the input requirements below and [upload them into a blob storage container](https://docs.microsoft.com/en-us/azure/cognitive-services/form-recognizer/build-training-data-set#upload-your-training-data). Single-page PDF is supported for batch redaction only. You need to convert multi-page PDF files to images with below format before labeling.
+  - Format must be JPG, JPEG, PNG, BMP or TIFF.
+  - File size must be less than 50 MB.
+  - Image dimensions must be between 50 x 50 pixels and 10000 x 10000 pixels.
+* Generate a Shared access (SAS) token and URL for FoTT to access the blob storage. Make sure you have selected all the options in “Permissions”.
```

Comment:
suggestion (non-blocking): list out all Permissions options (read, list, write, etc.)

Reply:
Added the list of permissions.